server/src/core/utilities.ts

```
@@ -455,6 +455,40 @@ function generatePlayerListPayload(connectionIDs: string[]) {
+  const keys = keyify(data);
+  for (const key of keys) {
+    result[key] = _.get(data, key);
+  }
+  return result;
+}
+
+// from: https://stackoverflow.com/a/47063174
+const keyify = (obj: any, prefix = ""): any =>
+  Object.keys(obj).reduce((res: any, el: any) => {
+    if (typeof obj[el] === "object" && obj[el] !== null) {
+      return [...res, ...keyify(obj[el], prefix + el + ".")];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)